src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -299,7 +299,7 @@ public class " + libName + @" {
 		}
 
 		[Test]
-		public void HybridAOT ()
+		public void HybridAOT ([Values ("armeabi-v7a;arm64-v8a", "armeabi-v7a", "arm64-v8a")] string abis)
```

Comment:
Should we also be including abi values for x86 and x86_64 here? Or are they covered by a different existing test.

Reply:
To give my two cents, I'd be OK with either option. Based on some quick local attempts on an x86_64 emulator, it seems that both x86 and x86_64 currently abort due to segmentation faults in hybrid AOT mode, so they are in roughly the same non-working category as armeabi-v7a. I could add them to the list of architectures that are rejected by the error message, if that sounds best?

Since the intersection of usage of hybrid AOT with x86 or x86_64 ABIs in real-world apps is likely very small, I'd also be OK leaving those out of this PR.

Reply:
ok lets leave them out for now

Reply:
Small correction to be complete: My x86_64 emulator was apparently in an odd state. After running `-wipe-data` on it, hybrid AOT with the x86 ABI now runs successfully, but with the x86_64 ABI, it still aborts. I tested and saw the same results with both with Visual Studio 2019 version 16.9 Preview and Visual Studio 2019 version 16.7.

So, for what it's worth, it seems that only x86_64 is non-working like armeabi-v7a.